sql/pg_net--0.7.3--0.8.0.sql

```diff
@@ -0,0 +1,17 @@
+create or replace function net.worker_restart() returns bool as $$
+  select pg_reload_conf();
+  select pg_terminate_backend(pid)
+  from pg_stat_activity
+  where backend_type ilike '%pg_net%';
+$$
+security definer -- needs SDF because of pg_terminate_backend
+language sql;
+
+grant all on schema net to PUBLIC;
+grant all on all tables in schema net to PUBLIC;
```

Comment:
This should be `usage` - we don't want people creating objects in the `net` schema